UnitTests/UnitTests.UWP/Extensions/Test_FrameworkElementExtensions.cs

```diff
@@ -0,0 +1,50 @@
+using Microsoft.Toolkit.Uwp;
+using Microsoft.Toolkit.Uwp.UI;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Threading.Tasks;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Markup;
+
+namespace UnitTests.Extensions
+{
+    public class Test_FrameworkElementExtensions : VisualUITestBase
+    {
+        [TestCategory("FrameworkElementExtensions")]
+        [TestMethod]
+        public async Task Test_Ancestor_WeakReference()
+        {
+            // Need to sim loading the control, and not just load the XAML via XamlReader
+            var view = App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var treeRoot = XamlReader.Load(
+           @"<Page
+                    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+                    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""
+                    xmlns:controls=""using:Microsoft.Toolkit.Uwp.UI.Controls""
+                    xmlns:ui=""using:Microsoft.Toolkit.Uwp.UI"">
+
+                    <Grid x:Name=""OuterGrid"" Width=""200"" Background=""Khaki"">
+                        <Button x:Name=""InnerButton"" ui:FrameworkElementExtensions.AncestorType=""Grid"" />
+                    </Grid>
+
+                </Page>") as Page;
+
+                Assert.IsNotNull(treeRoot, "Could not load XAML tree.");
+                await SetTestContentAsync(treeRoot);
+
+                // Need to simulate loading the control (and not just rely upon XamlReader.Load)
```

Comment:
Shouldn't this be handled by the SetTestContentAsync above? as that actually puts the content from `treeRoot` into the actual visual tree of the test app.

Reply:
Did you mean "... _EnqueueAsync_ above"?  

In trying to repro the original bug when a XAML control is actually **loaded**, I found that just using spinning up the XAML with `XamlReader.Load` wasn't working, hence the code displayed here being off the beaten path a little bit, and specifically navigating to it.

Reply:
`await SetTestContentAsync(treeRoot);` is setting the XAML from XamlReader to the test app's content in the Visual Tree, so it should be loading the elements already... so I'd be surprised you'd have to force them to load again.